src/core/errorHandler.js

```diff
@@ -64,7 +64,7 @@ export type ErrorHandlerType = typeof ErrorHandler;
  * The decorator will assign an ErrorHandler instance to the errorHandler
  * property.
  *
- * For convenience, you can use withErrorHandling() instead which will
+ * For convenience, you can use withRenderedErrorHandler() instead which will
```

Comment:
Would you also like to get rid of the example? Or is it just that sentence that I should rephrase?